sdk/python/feast/repo_config.py

```diff
@@ -190,6 +199,39 @@ def _validate_offline_store_config(cls, values):
 
         return values
 
+    @root_validator(pre=True)
+    def _validate_feature_server_config(cls, values):
+        # Having no feature server is the default.
+        if "feature_server" not in values:
+            return values
```

Comment:
Instead of returning values here, can you create an empty `values["feature_server"]` dict? (e.g. see _validate_online_store_config how they do this). This way, it's easier to read config in other places. For example, we'll know that feature_server always exists (and enabled will be False by default), even if the entire section is missing from the yaml file